pkg/webui/console/components/device-data-form/index.js

```diff
@@ -36,44 +36,82 @@ import PropTypes from '../../../lib/prop-types'
 import m from './messages'
 import validationSchema from './validation-schema'
 
-@bind
 class DeviceDataForm extends Component {
   constructor(props) {
     super(props)
 
-    const { initialValues } = this.props
+    const {
+      initialValues: { supports_join, ids, root_keys = {} },
+    } = this.props
+
+    const external_js = !Boolean(Object.keys(root_keys).length)
```

Comment:
This will make the external checkbox checked by default in the create form. Is this desired?

Reply:
@johanstokking

Reply:
Yes. I was asking for that change but then realized that this should become the normal way to onboard devices; the keys should already be in a Join Server